boot/freeldr/freeldr/CMakeLists.txt

```diff
@@ -282,9 +282,9 @@ if(MSVC)
         add_target_link_flags(freeldr_pe "/ignore:4078 /ignore:4254 /DRIVER")
         add_target_link_flags(freeldr_pe_dbg "/ignore:4078 /ignore:4254 /DRIVER")
     else()
-        add_target_link_flags(freeldr_pe "/ignore:4078 /ignore:4254 /DRIVER /FIXED /FILEALIGN:0x200 /ALIGN:0x200")
+        add_target_link_flags(freeldr_pe "/ignore:4078 /ignore:4254 /DRIVER /FIXED /FILEALIGN:512 /ALIGN:512")
         add_linker_script(freeldr_pe freeldr_i386.msvc.lds)
-        add_target_link_flags(freeldr_pe_dbg "/ignore:4078 /ignore:4254 /DRIVER /FIXED /FILEALIGN:0x200 /ALIGN:0x200")
+        add_target_link_flags(freeldr_pe_dbg "/ignore:4078 /ignore:4254 /DRIVER /FIXED /FILEALIGN:512 /ALIGN:512")
```

Comment:
Is that a regression of their linker?

Reply:
No, it's MSBUILD complaining, it must be extracting the value from the flag:
`2021-02-03T15:13:57.1115193Z C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(1084,9): error MSB4030: "0x200" is an invalid value for the "SectionAlignment" parameter of the "Link" task. The "SectionAlignment" parameter is of type "System.Int32". [D:\a\reactos\reactos\build\boot\freeldr\freeldr\freeldr_pe.vcxproj]`

Reply:
lmfao 🤣 🤣